gp-populate-anything/gppa-force-gppa-on-save-and-continue.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Gravity Perks // Populate Anything // Force GPPA value over a Save & Continue Value
+ * https://gravitywiz.com/documentation/gravity-forms-populate-anything/
+ */
+add_filter( 'gform_incomplete_submission_post_get', function( $submission_json, $resume_token, $form ) {
+	// Update "29" to your form ID, and "1" to your field ID
+	$target_form_id  = 29;
+	$target_field_id = 1;
+	static $_gppa_forcing_hydration;
+	if ( $form['id'] == $target_form_id && ! $_gppa_forcing_hydration ) {
```

Comment:
Should there be a check for the field ID here? You might need to introduce a foreach as being able to specify multiple fields would be nice.

Reply:
Gotcha!